packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -100,17 +101,20 @@ export const defaultSweepGracePeriodMs = 1 * oneDayMs; // 1 day
  */
 export interface GCFeatureMatrix {
 	/**
+	 * Legacy - new containers would get gcGeneration instead (if anything)
+	 *
 	 * The Tombstone Generation value in effect when this file was created.
 	 * Gives a way for an app to disqualify old files from GC Tombstone enforcement.
 	 * Provided via Container Runtime Options.
 	 */
 	tombstoneGeneration?: number;
+
 	/**
-	 * The Sweep Generation value in effect when this file was created.
+	 * The GC Generation value in effect when this file was created.
 	 * Gives a way for an app to disqualify old files from GC Sweep.
```

Comment:
Maybe point to the documentation for `gcGenerationOptionName` above so both are in sync.